[2025-09-09T14:58:06.507+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db scheduled__2025-09-05T04:35:00+00:00 [queued]>
[2025-09-09T14:58:06.532+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db scheduled__2025-09-05T04:35:00+00:00 [queued]>
[2025-09-09T14:58:06.534+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2025-09-09T14:58:06.609+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_sql_db> on 2025-09-05 04:35:00+00:00
[2025-09-09T14:58:06.644+0000] {standard_task_runner.py:60} INFO - Started process 503 to run task
[2025-09-09T14:58:06.652+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'banvic_etl', 'extract_sql_db', 'scheduled__2025-09-05T04:35:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/bavic_dag.py', '--cfg-path', '/tmp/tmpceb9rfm6']
[2025-09-09T14:58:06.660+0000] {standard_task_runner.py:88} INFO - Job 130: Subtask extract_sql_db
[2025-09-09T14:58:07.081+0000] {task_command.py:423} INFO - Running <TaskInstance: banvic_etl.extract_sql_db scheduled__2025-09-05T04:35:00+00:00 [running]> on host 9948058d4968
[2025-09-09T14:58:07.793+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Paulo' AIRFLOW_CTX_DAG_ID='banvic_etl' AIRFLOW_CTX_TASK_ID='extract_sql_db' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T04:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-05T04:35:00+00:00'
[2025-09-09T14:58:07.938+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:58:08.206+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:58:08.321+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:58:08.490+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-09T14:58:08.590+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=banvic_etl, task_id=extract_sql_db, execution_date=20250905T043500, start_date=20250909T145806, end_date=20250909T145808
[2025-09-09T14:58:08.764+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-09T14:58:08.898+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
